base/_variables.scss

```diff
@@ -1 +1,15 @@
-// @file variables.scss
+// Variables throughout the project
```

Comment:
I think that yes, there are some that deserve to be used more globally. Something like font-family could be used in many other places other than just the `typography.scss` file. We have all colors broken out into its own file, so now I think we need a place to put the rest of the global variables. You had an empty file in the scaffold before, I was trying to put a few in there that made sense.